examples/probabilistic_matrix_factorization.py

```diff
@@ -0,0 +1,57 @@
+import edward as ed
```

Comment:
I recommend styling the header/imports similar to the other examples scripts. In particular, include `#!/usr/bin/env python`, then add a short docstring written below via `"""[text]"""`, then add `from __future___ ...`, and alphabetize your import statements.